### ISSUE TITLE
Node Tooltip QoL Suggestion

### ISSUE THREAD

**Issue by Korrasera** [DestinyItemManager/DIM#8797]
### Proposed change

In the item popup, hovering over a node (perks, barrels, mags, sights, etc) will bring up a small tooltip with information on the node. This tooltip is anchored to the node itself, displaying just to the side of the node. Example:

![tooltip](https://user-images.githubusercontent.com/118650/197298768-f1aedcac-cd31-4a26-a9f2-562bdc38b3cd.png)

Anchoring the tooltip to the node means that the tooltip will obscure other nodes on the item, requiring you to move off of the item to dismiss the tooltip before you can look at a node that was obscured by the tooltip.

I propose changing the anchor so that tooltips are anchored the larger item pane itself, so that the never coverup the selectable nodes that they're tooltipping for. If they were anchored at the bottom of the item pane that would preserve horizontal space, but anchoring them to the right side of the pane would suffice for any desktop users.

### How does this fit into your workflow?

This is a minor quality of life improvement. Simply put, it's irritating ux to have to shake the mouse around to jump between different nodes on an item. Having to click off or make an intentional move to get rid of an overlay that is useful but gets in the way is a frustrating experience. I'd like to be able to evaluate all of the nodes on an item at a glance, so that I just need to mouse over one after the other to see all of the additional details on an item.

In short, it'll make perk switching on items feel less irritating.

**Comment by Revadike** [comment]
I find it hard to understand exactly what you mean. I'm trying to have the UX you're describing, but I can't really get it to happen?

**Comment by Korrasera** [comment]
What I'm suggesting is like this: In the image above, the box that says Alloy Magazine should instead be anchored so that it appears to the left, right, or bottom of the weapon pane itself and doesn't display laid over the weapon pane, obscuring the other nodes available on the weapon.

To put that differently, look at the way the actions panel is anchored to the left size of the weapon pane. What I'm saying is that the node tooltip should work like that, appearing to the side instead of being layered over the window pane itself.

**Comment by bhollis** [comment]
You can also click on the node itself to get the description inline.
